src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -27,74 +51,588 @@ const ActorInit En_Dns_InitVars = {
+        func_8092D5E8(this, globalCtx);
+        SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+        func_8092C5C0(this);
+        return;
+    }
+
+    this->actionFunc(this, globalCtx);
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    func_8092C934(this);
+    func_8092C86C(this, globalCtx);
+    Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 30.0f, 12.0f, 0.0f, 4);
+    func_8013C964(&this->actor, globalCtx, 80.0f, 40.0f, 0, this->unk_2C6 & 7);
+    Actor_SetHeight(&this->actor, 34.0f);
+    func_8092C6FC(this, globalCtx);
+    func_8092C5C0(this);
+}
+
+s32 func_8092D954(s16 arg0, s16 arg1, Vec3f* arg2, Vec3s* arg3, s32 arg4, s32 arg5) {
+    Vec3f sp74;
+    Vec3s sp6C;
+    MtxF sp2C;
+
+    SysMatrix_MultiplyVector3fByState(&D_801D15B0, &sp74);
+    SysMatrix_CopyCurrentState(&sp2C);
+    func_8018219C(&sp2C, &sp6C, 0);
+    *arg2 = sp74;
+
+    if (arg4 == 0) {
+        if (arg5 != 0) {
+            sp6C.z = arg0;
+            sp6C.y = arg1;
+        }
+        Math_SmoothStepToS(&arg3->x, sp6C.x, 3, 0x2AA8, 0xB6);
+        Math_SmoothStepToS(&arg3->y, sp6C.y, 3, 0x2AA8, 0xB6);
+        Math_SmoothStepToS(&arg3->z, sp6C.z, 3, 0x2AA8, 0xB6);
+    } else {
+        arg3->x = sp6C.x;
+        arg3->y = sp6C.y;
+        arg3->z = sp6C.z;
+    }
+
+    return 1;
+}
+
+s32 EnDns_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnDns* this = THIS;
+
+    this->unk_1E4[limbIndex] = *dList;
+    *dList = NULL;
+    return false;
+}
+
+void EnDns_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnDns* this = THIS;
+    s32 pad;
+    s32 phi_v1;
+    s32 phi_v0;
+
+    if (this->unk_2C6 & 0x10) {
+        phi_v1 = 1;
+        phi_v0 = 0;
+    } else {
+        phi_v1 = 0;
+        if (this->unk_2C6 & 0x20) {
+            phi_v0 = 1;
+        } else {
+            phi_v0 = 0;
+        }
+    }
+
+    if (limbIndex == 2) {
+        func_8092D954(this->unk_2CC, this->unk_2CE + this->actor.shape.rot.y, &this->unk_218, &this->unk_224, phi_v1,
+                      phi_v0);
+        SysMatrix_InsertTranslation(this->unk_218.x, this->unk_218.y, this->unk_218.z, MTXMODE_NEW);
+        Matrix_Scale(this->actor.scale.x, this->actor.scale.y, this->actor.scale.z, MTXMODE_APPLY);
+        Matrix_RotateY(this->unk_224.y, MTXMODE_APPLY);
+        SysMatrix_InsertXRotation_s(this->unk_224.z, MTXMODE_APPLY);
+        SysMatrix_InsertZRotation_s(this->unk_224.x, MTXMODE_APPLY);
+    }
+
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+    gSPDisplayList(POLY_OPA_DISP++, this->unk_1E4[limbIndex]);
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void EnDns_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    static Gfx* D_8092DE1C[] = { D_060028E8, D_06002968, D_060029E8, D_06002968 };
```

Comment:
Those looks like textures, not Dlists
```suggestion
    static TexturePtr D_8092DE1C[] = { D_060028E8, D_06002968, D_060029E8, D_06002968 };
```